libs/constants.py

```diff
@@ -1,11 +1,24 @@
 NO_INTERVENTION = 0
+FLATTEN = 1
+FULL_CONTAINMENT = 2
+SOCIAL_DISTANCING = 3
 
+INTERVENTIONS = [
```

Comment:
I think this is a good use case for the [enum](https://docs.python.org/3/library/enum.html#creating-an-enum). 

```
import enum
class Intervention(enum.Enum):
    NO_INTERVENTION = 0
    FLATTEN = 1
    FULL_CONTAINMENT = 2
    SOCIAL_DISTANCING = 3
```

Then you can access it like:
```
Intervention.SOCIAL_DISTANCING.value  # 1
Intervention.SOCIAL_DISTANCING.name  # "SOCIAL_DISTANCING"

```

Reply:
ya sorry was being lazy